news/_posts/releases/2020-01-21-prestashop-1-7-6-3-maintenance-release.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.3"
+subtitle: "Maintenance version of the 1.7.6.x branch"
+date:   2020-01-21 09:10:11
+authors: [ PrestaShop ]
+image: /assets/images/2020/01/1763-image.jpg
+icon: icon-checked
+tags:
+ - 1.7
+ - 1.7.6.x
+ - version
+ - patch
+ - releases
+---
+
+After a few months dedicated to fixing all regressions found on the previous 1.7.6 versions, PrestaShop 1.7.6.3 is finally available! 
+
+![1.7.6.3 is available!](/assets/images/2020/01/1763-banner.jpg)
+
+This maintenance release fixes *16 issues* reported on versions 1.7.6.0, 1.7.6.1, and 1.7.6.2.
+
+Obviously, we suggest upgrading your shop quickly in order to benefit from those fixes - and do not forget to backup before. ;-)
+
+Reminder: our 1-Click Upgrade module’s latest version is v4.10.1 ([read here](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.10.1)), do not forget to upgrade it as well.
+
+
+## Regression fixes
+
+Below are listed all the regression issues we fixed in this version:
```

Comment:
```suggestion
Below are listed all the regression issues fixed in this version:
```
remove "we".
this is a collaborative work and part of it might not have been done only by the PrestaShop employees, but from/with external contributors too.